package/Dockerfile

```diff
@@ -36,6 +36,7 @@ ENV TINI_VERSION v0.18.0
 ENV TELEMETRY_VERSION v0.5.16
 ENV DOCKER_MACHINE_LINODE_VERSION v0.1.8
 ENV LINODE_UI_DRIVER_VERSION v0.3.0
+ENV DOCKER_MACHINE_HARVESTER_VERSION v0.3.2
```

Comment:
If the version is going to be here and in `machinedriver_data.go`, please add a comment here to indicate that the version should be updated there as well.

Reply:
@thedadams  I have added the comment at both places. The pr is ready for re-review. Thanks a lot for the input!